extensions/email/src/main/java/org/mule/extension/email/api/Email.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.api;
+
+import org.mule.runtime.extension.api.annotation.Parameter;
+
+/**
+ * Email message representation, holding its {@link EmailContent} and {@link EmailAttributes}
+ *
+ * @since 4.0
+ */
+public class Email
+{
+
+    @Parameter
+    private EmailContent content;
+
+    @Parameter
+    private EmailAttributes attributes;
+
+    public Email()
+    {
+    }
+
+    public Email(EmailContent emailContent, EmailAttributes attributes)
+    {
+        this.content = emailContent;
+        this.attributes = attributes;
+    }
+
+    public EmailContent getContent()
+    {
+        return content;
+    }
+
+    public EmailAttributes getAttributes()
```

Comment:
why having this EmailAttributes at all? If this class represents the email and now we say that the subject is just as much part of the email as the content, then why aren't those attributes fields of this class?